fv3config/_asset_list.py

```diff
@@ -172,7 +172,7 @@ def write_asset(asset, target_directory):
     if not os.path.exists(os.path.dirname(target_path)):
         os.makedirs(os.path.dirname(target_path))
     if copy_method == "copy":
-        get_file(source_path, target_path)
+        filesystem.get_file(source_path, target_path, cache=True)
```

Comment:
This is the only place I've added caching, which I think is sufficient. Let me know if that's not the case.